cpp/tests/table/table_tests.cu

```diff
@@ -14,18 +14,138 @@
  * limitations under the License.
  */
 
+//#include <cudf/column/column.hpp>
```

Comment:
If possible, this file should be made a .cpp. I don't see any device code in here, so I don't think it should be a problem.